src/main/drivers/pwm_mapping.h

```diff
@@ -101,6 +101,12 @@ typedef enum {
   PWM_PF_OUTPUT_PROTOCOL_MULTISHOT = (1 << 5)
 } pwmPortFlags_e;
 
+typedef enum {
```

Comment:
May be adding the brushed motors here as an additional protocol is more clean if we restructure everything anyhow?